Makefile

```diff
@@ -46,12 +58,32 @@ run-mybinder: build-mybinder # run the mybinder jupyter image
 ## Dockerhub image
 ##
 .PHONY: build-hub
-build-hub: ## Build dockerhub image
-	docker build -t freeandfair/sysmlv2-jupyter:$(sysml_release) -f Dockerfile.hub --build-arg RELEASE=$(sysml_release) .
+build-hub: ## build DockerHub image
+	docker buildx build --platform $(DOCKER_PLATFORM) -t $(DOCKER_ORGANIZATION)/$(DOCKER_IMAGE):$(sysml_release) -f Dockerfile.hub --build-arg RELEASE=$(sysml_release) .
```

Comment:
I have had to learn `buildx` recently due to other work too.  A whole new world of perhaps-over-engineered-abstraction...